src/utils/browser.ts

```diff
@@ -5,3 +5,7 @@ export const isSupportIsolation = (browserName: string, browserVersion = ""): bo
 
     return browserName === "chrome" && Number(browserVersionMajor) >= MIN_CHROME_VERSION_SUPPORT_ISOLATION;
 };
+
+export const isRunInBrowserEnv = (): boolean => {
+    return typeof window !== "undefined" && typeof window.document !== "undefined";
+};
```

Comment:
What if i mocked "globalThis.window" in node.js environment?
There are more reliable checks. I like this one, with `global` and `window` equality to `this` (i think, we better should compare with `globalThis`): https://stackoverflow.com/a/31090240

Reply:
I realized today that it doesn't work correctly. It means I can't realize is `mock` method called in node env or browser if it is used outside of hook/suite/test. Example:
```
import {mock} from "testplane";

mock('axios');

it('foo bar', () => {...});
```

This file will be read in master and worker process, before it will be run in browser env. It means that `mock` will be called in test parsing stage. So I need knowledge from config in which env it starts. But it means I need add this logic to the global.

We already had `global.testplane`, but I don't want to pollute to this variable. So right now I remove this checker and `mock` and `unmock` calls to nothing in node env.

I will create ticket about this problem.